src/components/SkipLink.tsx

```diff
@@ -1,27 +1,19 @@
 import { useTranslation } from "next-i18next"
-import { Box } from "@chakra-ui/react"
-
-import { BaseLink } from "@/components/Link"
 
 import { MAIN_CONTENT_ID } from "@/lib/constants"
 
+import { BaseLink } from "../../tailwind/Link"
+
 export const SkipLink = () => {
   const { t } = useTranslation()
   return (
-    <Box bg="primary.base">
+    <div className="bg-primary">
       <BaseLink
         href={"#" + MAIN_CONTENT_ID}
-        lineHeight="taller"
-        position="absolute"
-        top="-12"
-        ms="2"
-        color="background.base"
-        textDecorationLine="none"
-        _hover={{ textDecoration: "none" }}
-        _focus={{ position: "static" }}
+        className="absolute -top-3 ms-2 leading-8 text-background no-underline hover:no-underline focus:static"
```

Comment:
Noticing that the `absolute` positioning is enough to hide this (with `focus:static` kicking in when the keyboard tab stops there.

We may be able to just remove `-top-3`, but if we want to keep it, we should change this to `-top-12` to match before, which slides this element 3rem up, and off the top edge of the screen. Right now it's only sliding it up 0.75rem.

Reply:
Ah! good catch, my bad. Thanks!